test/unit/org/apache/cassandra/index/sai/cql/BM25Test.java

```diff
@@ -1067,26 +1067,26 @@ private void assertRejectsBM25OnNonRegularColumn(String query, String columnType
 
     public final static Object[][] DATASET =
     {
-    { 0, "Climate", 5, "Climate change is a pressing issue. Climate patterns are shifting globally. Scientists study climate data daily.", 1 },
```

Comment:
This commit is for cleaning/refactoring, why not remove the last two IDE warnings in the test class, too?

Reply:
I am actually not sure about fixing the warnings. Removing the result type will make the method signature inconsistent. 
The second case is on purpose.